builder.go

```diff
@@ -338,20 +339,16 @@ func (e *Expression) Literal(v any) *Expression {
 	return e
 }
 
-func Var(s string) *Expression {
-	return Expr().Var(s)
+func Var(name string) *Expression {
+	return Expr().Var(name)
 }
 
-func (e *Expression) Var(v string) *Expression {
-	if len(v) == 0 {
+func (e *Expression) Var(name string) *Expression {
```

Comment:
please add checking if variable is valid. according to [abnf](https://github.com/unicode-org/message-format-wg/blob/7c00820a0462679eba696181c45bfadb43d2eedd/spec/message.abnf#L88-L95)

e.g. 
calling Var with argument "$var" should error/panic